app/src/main/java/uk/gov/onelogin/login/BiometricsOptInScreen.kt

```diff
@@ -0,0 +1,41 @@
+package uk.gov.onelogin.login
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import uk.gov.android.ui.theme.GdsTheme
+
+@Composable
+fun BiometricsOptInScreen() {
+    GdsTheme {
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Center,
+            modifier = Modifier
+                .padding(16.dp)
+                .fillMaxSize()
+                .verticalScroll(rememberScrollState())
+        ) {
+            Text(
+                text = "Placeholder",
```

Comment:
Placeholder?

Reply:
Ah this is because the actual screen will be built in the next ticket, so for now the screen is purely a placeholder to allow the routing to happen properly